core/modules/qproc/SConscript

```diff
@@ -6,38 +6,42 @@ import os
 
 standardModule(env, test_libs='log4cxx')
 
+
 # build kvmap.h
 # If you need to rebuild the map, use:
 # ./admin/bin/qserv-admin.py < admin/examples/testMap_generateMap
 def charArrayGenerator(filename):
     bSize = 70
-    with open(filename,"rb") as f:
+    with open(filename, "r") as f:
```

Comment:
I've added a comment. Also removed that `bSize` variable that was not used anywhere.